Library/Formula/folly.rb

```diff
@@ -0,0 +1,45 @@
+class Folly < Formula
+  desc "An open-source C++ library developed and used at Facebook."
+  homepage "https://github.com/facebook/folly"
+  url "https://github.com/facebook/folly.git", :revision => "6457c4429379011d12ee841e8325dddeebec538e"
```

Comment:
the build doesn't work on OS X, it has since been fixed in master, but a release hasn't been cut